core/src/main/java/org/everit/json/schema/internal/TemporalFormatValidator.java

```diff
@@ -0,0 +1,39 @@
+package org.everit.json.schema.internal;
+
+import org.everit.json.schema.FormatValidator;
+import org.everit.json.schema.Validator;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoField;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Base class for date and time format validators
+ */
+public class TemporalFormatValidator implements FormatValidator {
+    private DateTimeFormatter formatter;
+    private List<String> formats_accepted;
```

Comment:
Thanks for the update @zgyorffi .
Please declare the instance-level fields below the static members. Also, please use camelCase naming instead of under_score (so please rename `formats_accepted` to `formatsAccepted` both here and in the constructor).
Otherwise it looks good now.